reference/configuration/framework.rst

```diff
@@ -2009,7 +2061,10 @@ provider
 
 **type**: ``string``
 
-The service name to use as provider when the specified adapter needs one.
+Overwrite the default service name or DSN respectively, if you do not want to
+use what is configured as ``default_X_provider`` under ``cache``. See the
+description of the default provider setting above for the type of adapter
+you use for information on how to specify the provider.
```

Comment:
@nicolas-grekas do i understand correctly that this takes the same value as the default provider, depending on the adapter? so it could be DSN or service, depending on the adapter

Reply:
correct